src/base/rss/rss_feed.cpp

```diff
@@ -195,7 +194,8 @@ bool Feed::hasError() const
 void Feed::handleDownloadFinished(const Net::DownloadResult &result)
 {
     if (result.status == Net::DownloadStatus::Success) {
-        qDebug() << "Successfully downloaded RSS feed at" << result.url;
+        LogMsg(tr("RSS feed at '%1' is successfully downloaded. Start to parse it.")
```

Comment:
I am a bit rusty on English, but I think "Starting to parse it" is better. @LordNyriox any input for this?

Reply:
@sledgehammer999, sorry, I probably won't be able to fix it anytime soon. Can you do it yourself (even as your own commit instead)?

Reply:
OFFTOPIC.
@sledgehammer999, since some issues can't be easily reproduced by developers we really need some more details from affected users in their reports. Can we add Debug level in our log and duplicate our debug messages there? Also it would be useful if we handle libtorrent log messages as well.

Reply:
@glassez I will be releasing tomorrow. Will you be able to fix it by tomorrow?

On the offtopic: You mean as another log category? I guess so. We only need to look out for performance though. Writing too many loglines (which end up on disk) may freeze up the UI.

Reply:
@sledgehammer999: 

> I think "Starting to parse it" is better. 

:+1:

Reply:
>  I think "Starting to parse it" is better.

Fixed.